network-api/networkapi/wagtailpages/pagemodels/buyersguide/products.py

```diff
@@ -246,15 +247,28 @@ def get_votes(self):
         votes = [int(x) for x in self.vote_bins.split(",")]
         return votes
 
-    def get_most_voted(self):
+    def get_vote_average(self):
         votes = self.get_votes()
         vote_breakdown = {k: v for (k, v) in enumerate(votes)}
-        highest = max(vote_breakdown, key=vote_breakdown.get)
-        label = self.get_vote_labels()[highest]
+        total_votes = sum(vote_breakdown.values())
+        """If there's no votes, let the user know (also protects from division by zero errors)"""
+        if total_votes == 0:
+            label = self.get_vote_labels()[5]
```

Comment:
Instead of accessing the "no votes" label by index, could we store `("No votes", gettext("No votes"))` as a variable so both here and in `def get_vote_labels` can ref it? Just a little worried that something's gonna be off if we ever make changes to the label list.

Reply:
I have a second thought and wonder if `("No votes", gettext("No votes"))` needs to be added to the label list?

Reply:
I pulled it out and just made it a part of the zero check's return, that way it's isolated and doesn't have to run anything additional.